server/database/buildDummy.js

```diff
@@ -0,0 +1,48 @@
+const request = require("request");
```

Comment:
@BenNewman90 what is the difference between this file and the `build.js` cause both build the database, also could you add database build script in the package json ?

Reply:
I've deleted it. It shouldn't have been pushed. Apologies. I've added a build script to the json.